weed/shell/command_fs_merge_volumes.go

```diff
@@ -40,7 +44,7 @@ func (c *commandFsMergeVolumes) Help() string {
 	
 	This would help clear half-full volumes and let vacuum system to delete them later.
 
-	fs.mergeVolumes -toVolumeId=y [-fromVolumeId=x] [-apply] /dir/
+	fs.mergeVolumes [-toVolumeId=y] [-fromVolumeId=x] [-collection="*"] [-apply] [/dir/]
```

Comment:
How about changing the directory into a named parameter also, in a separate PR?

Reply:
OK, after you merged this I'll send another PR. Is it okay with you?